src/ipfsd-client.js

```diff
@@ -9,6 +9,7 @@ const daemonLog = {
   info: logger('ipfsd-ctl:client:stdout'),
   err: logger('ipfsd-ctl:client:stderr')
 }
+const rpcModuleLogger = logger('ipfsd-ctl:client:rpcModule:stdout')
```

Comment:
The `stdout`/`stderr` suffix is because we're (in theory) logging the output of an external process, that's not the case for this logger so it should be shortened:

```suggestion
const rpcModuleLogger = logger('ipfsd-ctl:client')
```